user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg

```diff
@@ -0,0 +1,61 @@
+#-------------------------------------------------#
+#### LDO Picobilical Frame PCB A1.3 definition ####
+#-------------------------------------------------#
+
+[mcu toolhead]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+restart_method: command
+##--------------------------------------------------------------------
+
+[include config/mcu_definitions/toolhead/LDO_Picobilical_Frame_vA1.3.cfg] # Do not remove this line
```

Comment:
Hello 
I noticed that the the path to the config file was different : 
[include config/mcu_definitions/toolhead/LDO_Picobilical_vA1.3.cfg] 
regards

Reply:
Or maybe this file: LDO_Picobilical_Frame_vA1.3.cfg hasn't been committed. I am trying to get this to work on my Voron v0 built using the LDO kit and I am getting the following error : 
`
Option 'aliases' in section 'board_pins picobilical_mcu' must have 2 elements

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted
`